### ISSUE TITLE
Fails to Mount USB

### ISSUE THREAD

**Issue by ghost** [yifanlu/usbmc#2]
Hey, I'm having mounting issues on startup. 
I'm using the launch release of Enso Henkaku and after adding "usbmc" to the config on ur0, the PSTV now powers on the USB device on startup (in my case, a USB micro SD adapter) but does not mount it as ux0 (uses the memory card instead). 
 
Also (note sure if this is relevant) but I've always needed to un-plug and re-plug this adapter into the PSTV, while in VitaShell, to get it to power on and mount (on regular Henkaku).

**Comment by jeremyj563** [comment]
Same issue here. It appears that usbmc is currently broken and unusable :-(

**Comment by GrassHydra** [comment]
You're not the only ones, "Burnt-Lasagna" and "jeremyj563" !
To cheer you up a bit, having usbmc on enso skips the part of having to unplug and replug the usb storage in Vita Shell. Power up the pstv, start Vita Shell, press X while hovering over "Mount uma0:" then press X on the highlighted "Mount USB ux0:" and that's it ! The days of having to get up from our warm confortable spot on our living room sofa for a well-deserved gaming session after our favourite TV show has ended are over ! Well... It depends on you what we want to sacrifice in order to make it happen...
More specific,
My pstv's HDMI image is "zoomed" beyond the borders of the tv with usbmc running right after the start-up. So I have to reset my screen resolution from "Sound & Display" which means that everytime I cold boot my pstv I have to do the "Sound & Display" trick again... Time-consuming...
The image looks distorted... So... I have no other option... But to leave this plugin for now hoping it gets a well-deserved update. I feel you, guys... You must have been as disappointed as I were... when we found out this was not the plugin we wanted.
Ensō and usbmc could have been the best first year anniversary gift for the pstv community...

**Comment by yifanlu** [comment]
Duplicate of #1

**Comment by shmanmino** [comment]
Hey everyone on this thread I’m experiencing an issue with usmbc which occurred right after I upgraded my PSTV vita memory card from 8gb to 16gb because of insufficient storage when I first nodded my PSTV I was using my 8gb for specifically Adrenaline and games and a 128 usb mass storage for the pstv itself. I have Enso installed and Usmbc mounted my usb stick at boot up when I inserted the memory card for the first time it asked me to transfer my internal data to the memory card. Then I mounted usb as ux0 and ran usmbc   It gave me an error that it couldn’t read the storage capacity it wasn’t able to mount. I am Able to mount the USB through vita shell and it works just fine the odd thing I don’t have to replug it every time that it boots up. I would appreciate it if anyone can help me thank you in advance

**Comment by GrassHydra** [comment]
Sorry for the late response. I've lost my pstv to a power surge apparently. One day I couldn't boot it anymore. Note to gamers : Do not crack the shell like I did just to expose the tiny fragile thing to moisture and dust that may damage the components. You have been warned. So... Back to your problem, shmanmino, a gamer in need is a gamer indeed. Simply plug in your original memory card and do a full format. Lose everything now, gain everything later. Try to backup only what you truly need. Take this with a pinch of salt. It's a lengthy process. Assuming you're on 3.60, you'll lose enso. Install Enso then install usbmc and do the switch. 128gb I assume. Put everything in there. Here's the twist : Once you have a working 128gb storage medium for your microconsole, do let chkdsk run wild on your pc. Apparently TheFlow's masterpiece, VitaShell, has this flaw : If you put the 128 inside Windows in your case it will be dirty. So clean it with : chkdsk F: /f . Here F is your drive letter and /f means cleaning the volume. Then use VPKtool (that unzips VPKs to game folders and game files). Clearly this is easier to do if you plan on getting the most out of your storage medium. Get everything back in there meaning everything you backed up earlier and you're good to go.

**Comment by shmanmino** [comment]
Thank you so much for getting back to me regarding my issue I will definitely give this a try because most likely my theory is that messing around with the USB stick and results to some failure I do have an additional 256 GB sticks which has been just formatted to Exfat I will transfer all the data from one to the other hopefully that will work and if not I will start it’s life from scratch LOL

**Comment by shmanmino** [comment]
I’m sorry to hear about your pstv though Maybe you can find one online brand new for cheap which will automatically come on the Lowest Fw

**Comment by GrassHydra** [comment]
To shmanmino : Firstly it's been some time since i didn't used my github account so thank you for reminding me my forgotten username. Secondly you don't need to be sorry for the past but for the future. Things come and go, I feel your pain. Probably it's empathy but we gamers know each other better because of our common interests. I've been up all that night trying to make things right for myself. I felt bad for letting go of my 5.05 modded ps4 for a douchebag. I only got 10 $ profit. He didn't knew it because I haven't told him. Frankly I had no idea what he was planning to do with it. I bought it for 186 $ with 1 controller and 2 games on 5.05 stock. I helped this nice guy who initially sold me this diamond in the rough because he had this nice BMW with a broken catalyst. Sucked for him because I admired him for his sincerity. One night the douchebag came to me looking creepy as heck and he fed me this sob story that you couldn't have resisted not falling for. He was a gamer. Apparently he had all the bells and whistles with him that made him look totally legit. He even told me he would come back later to buy my enso 3g. He lied. I never heard from him again. Imagine him gaining 200 $ just for exploiting me as a human the same way we know who likes to exploit Sony. I failed to learn the hard to understand lesson of exploits. In real life. 3 hours later you came asking for help. I was late. I felt sorry even for this. But now He Who Lives In Heavens read my desperate thoughts from across the Universe. I got to own this slim which is a 802.11 ac improvement over my last console and I'm delighted with it. Imagine He gave me my life back. The very thing I simply sacrificed back then in the douchebag's favor came back to me twice as fast than a boomerang with a sleeker design. I guess He knows better what offer He made me. I took it nonetheless. His ways are just right. Now if you've been following this throughoutly you might have noticed how much I gained in a material form and how much I suffered psychically for my actions. He showed me no debt He has remains unpaid. I asked, He answered. Back to the pstv, you tell me why He chose in His Might to brick my pstv. Thirdly, and note it down : "It feels right to choose the bright path. It only feels better when you don't".
